docs/pages/component/methods.mdx

```diff
@@ -64,6 +64,31 @@ Future:
 - Will support more formats in the future through options
 - Will support custom directory and file name through options
 
+### `capture`
+
+<PlatformsList types={['iOS', 'Android']} />
+
+`capture(): Promise<void>`
```

Comment:
Is it possible to manage the generate file here ?
2 solutions:
- allows to give a path / file path in parameters
- give the generated file path in promise result

Because, here, I cannot know where the file is generated